### PR TITLE
Feature/views and templates

### PR DESCRIPTION
**Pull request by ruands**

### CODE REVIEW

**Review by qoda — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+Content types
+-------------
+
+FAQ
+***
+
+FAQ introduces 2 fields called `Question` and `Answer`.
+
+1.  Question is a text field.
+2.  Answer is a rich-text field of arbitrary length.
```
Comment (hedleyroos) [inline README.rst:21]: You need an empty line at the end of rst files.

File: faq/templates/faq/inclusion_tags/faq_detail.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% extends "jmbo/inclusion_tags/modelbase_detail.html" %}
+{% load ultracache_tags %}
+
+{% block content %}
+    {% ultracache 3600 "object-detail-inclusion" object.id object.modified request.page %}
```
Comment (hedleyroos) [inline faq/templates/faq/inclusion_tags/faq_detail.html:5]: request.page does not belong in here. FAQ's don't have pagination.

File: faq/templates/faq/inclusion_tags/faq_detail.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        {% with object.content_pages as content_pages %}
+            {% autopaginate content_pages 1 %}
+            <div class="content">
+                <a name="jmbo-faq"></a>
+                {{ content_pages.0|safe }}
```
Comment (hedleyroos) [inline faq/templates/faq/inclusion_tags/faq_detail.html:10]: Or did you paginate faqs? That's redundant - remove it.

File: faq/templatetags/faq_inclusion_tags.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.inclusion_tag("faq/inclusion_tags/faq_detail.html")
```
Comment (hedleyroos) [inline faq/templatetags/faq_inclusion_tags.py:6]: Redundant. Remove the entire templatetags directory. I must still remove it from jmbo-post.

File: faq/tests/test_views.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+            answer="Yes, it is",
+            state="Published"
+        )
+        obj.sites = [1]
+        cls.faq = obj
+
+    def test_pagination(self):
+        response = self.client.get(self.faq.get_absolute_url())
+
+    @ classmethod
```
Comment (hedleyroos) [inline faq/tests/test_views.py:26]: There is a space after the `@`. Should not be there.

**Pull request closed without merging**